### PR TITLE
Mobile version skeleton

### PR DESCRIPTION
**Pull request by Kambuchi**
**--Added header and hero sections**
Generate and basic structure, filled with the two first part: _**header and hero**_
**Header**
Added a navigation bar
Added a ≡ for toggle menu
Added a logo for the mobile version
**Hero**
Added and h1 title and descriptive paragraph
Added a button for hero

**--Added assets**
Created a fonts folder
Created a img folder

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+  .nav-list {
+    display: none;
+  }
+
+  .logo-nav {
+    display: flex;
+  }
+
+  .toggleNav {
```
Comment (henatan99) [inline style.css:98]: - [ ] The hamburger button should be placed on the right. Please make the necessary changes in the styling. Kindly refer to the below screenshot images to compare with the original Figma design. 
## Figma design
Screenshot (6)

## Your design
Screenshot (4)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+}
+
+.content {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (henatan99) [inline style.css]: - [ ] Please avoid the flex-wrap property as it renders a design different from the original Figma design.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  justify-content: space-evenly;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+nav li {
+  list-style: none;
+}
```
Comment (henatan99) [inline style.css:56]: - [ ] Please add color similar to the original design font color in the nav list items.

File: index.html [hunk 4]
```diff
@@ -4,9 +4,30 @@
...
+                <ul class="nav-list">
+                    <li><a href="#">Hello</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+        <section class="hero">
+            <h2>Hector<br>Pereira</h2>
```
Comment (henatan99) [inline index.html]: - [ ] Please make this tag an 'h1' tag. It is always advisable to follow descending order of heading tags on pages, starting from the 'h1' tag.

File: index.html [hunk 5]
```diff
@@ -4,9 +4,30 @@
...
+            <h3>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</h3>
```
Comment (henatan99) [inline index.html]: - [ ] This section is a paragraph and the proper semantic tag is p-tag. Kindly change it to p-tag.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+.hero button {
+  width: 288px;
+  height: 50px;
+  background-color: #1a2236;
+  color: white;
+  align-self: center;
+  margin-top: 25px;
+}
```
Comment (henatan99) [inline style.css:74]: - [ ] The hero button does not have the proper styling as the original design. Please see the attached screenshot for button styling given in the original design and apply it in your design for different states(enabled, hover, pressed, disabled if possible). 
Screenshot (5)

File: style.css [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+@media only screen and (max-width: 600px) {
```
Comment (henatan99) [inline style.css:82]: - [ ] OPTIONAL comment: Always practice a mobile-first approach.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+  font-style: normal;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  color: #1a2236;
+}
+
+h2 {
+  font-family: 'Playfair Display', sans-serif;
```
Comment (henatan99) [inline style.css:19]: - [ ] Please use a font family that resembles the original design for this font.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+  text-decoration: none;
+}
+
+nav li {
+  list-style: none;
+}
+
+.hero {
+  padding: 0 5vh 5vh 5vh;
+  margin: 0;
```
Comment (henatan99) [inline style.css]: - [ ] Please give the hero section a proper margin, referring to the original design.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi Kambuchi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

Also, please make changes to the readme template. You just left it with its default state. This is mandatory! 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It is a best practice to use a mobile-first approach.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (DeliceLydia) [reply style.css]: This was addressed perfectly your design is similar to the given design well done

File: style.css (hunk 3 above)
Comment (DeliceLydia) [reply style.css:56]: Well addressed!

File: index.html (hunk 4 above)
Comment (DeliceLydia) [reply index.html]: Well addressed the issue!

File: index.html (hunk 5 above)
Comment (DeliceLydia) [reply index.html]: Well done on using p tag as requested

File: style.css (hunk 9 above)
Comment (DeliceLydia) [reply style.css]: Well done!

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉🎉

Congratulations @Kambuchi! 🎉 Your project is complete!  There is nothing else to say other than... am super honored to approve it's time to merge and move on to the next project.


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**